jjb/anax/Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
+    agent any
+    stages {
+	stage('Install Dependencies'){
+	    steps{
+		sh 'echo "Installing dependencies"'
+		sh '''
+			#!/usr/bin/env bash
+                        apt-get update -qq && apt-get install -y \
```

Comment:
This looks like a container script. Our builders run as the "jenkins" user, not as root, so installation will require `sudo`.